### PR TITLE
Feature/#83 project details api update

### PR DESCRIPTION
**Pull request by lkolacz**

### CODE REVIEW

File: backend/unpp_api/apps/partner/models.py [hunk 1]
```diff
@@ -37,13 +37,21 @@ class Partner(TimeStampedModel):
...
-    registration_number = models.CharField(max_length=255, null=True, blank=True)
-    country_presents = ArrayField(
+    # hq information
+    country_presence = ArrayField(
         models.CharField(max_length=2, choices=WORKING_LAGNUAGES_CHOICES),
         default=list,
         null=True
     )
     staff_globally = models.CharField(max_length=3, choices=STAFF_GLOBALLY_CHOICES, null=True, blank=True)
+    # country profile information
```
Comment (elyk) [inline backend/unpp_api/apps/partner/models.py]: ?

Comment (lkolacz) [reply backend/unpp_api/apps/partner/models.py]: below information will be used to country profile and for HQ, there will be empty

Comment (elyk) [reply backend/unpp_api/apps/partner/models.py]: I'm confused on why we're setting them to None, though?

Comment (lkolacz) [reply backend/unpp_api/apps/partner/models.py]: now I got You...
I wasn't sure about them (then)... now they are defined in next pull request (according to mockups and doc field spec) ;-)
Sorry for bringing misunderstand!

**Pull request merged**